src/applications/vaos/new-appointment/components/ConfirmationPage/ConfirmationDirectScheduleInfoV2.jsx

```diff
@@ -38,19 +55,23 @@ export default function ConfirmationDirectScheduleInfoV2({
         <strong>We’ve scheduled and confirmed your appointment.</strong>
         <br />
         <div className="vads-u-margin-y--1">
-          <Link
-            to="/"
+          <VaLink
```

Comment:
Let's use `va-link` for consistency
```suggestion
          <va-Link
```